live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -8,24 +8,28 @@ const StyledWelcomeSectionContainer = styled.div`
   flex-direction: row;
   justify-content: center;
   margin: 1.3em;
-  @media ${max.tablet} {
+  @media ${max.tabletLg} {
     flex-direction: column;
     align-items: center;
   }
 `;
 
 const StyledRaceContainer = styled.div`
```

Comment:
Could you please add this styling here to reduce the width at this screensize? Since it's like almost touching the edges 

```
  @media ${max.tabletLg} {
    width: 40em;
  }
```

<img width="955" alt="Screenshot 2023-01-23 at 6 41 52 PM" src="https://user-images.githubusercontent.com/59738880/214176939-624d8561-bbeb-4a05-8141-ba07d72ba426.png">